drivers/backup/portworx/portworx.go

```diff
@@ -1113,6 +1113,33 @@ func (p *portworx) GetBackupUID(ctx context.Context, backupName string, orgID st
 	return "", fmt.Errorf("backup with name '%s' not found for org '%s'", backupName, orgID)
 }
 
+func (p *portworx) GetRecentScheduleBackupName(ctx context.Context, backupName string, orgID string) (string, error) {
+	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
+		OrgId: orgID,
+		Name:  backupName,
+		Uid:   "",
+	}
+	resp, err := p.InspectBackupSchedule(ctx, backupSchInspectRequest)
+	if err != nil {
+		return "", err
+	}
+	schBackupName := resp.GetBackupSchedule().GetBackupStatus()["interval"].GetStatus()[0].GetBackupName()
```

Comment:
Change it from `schBackupName` to `scheduledBackupName`

Reply:
I am considering changing it to recentScheduleBackupName

Reply:
Ok. Makes sense. Or `latestScheduledBackupName`

Reply:
Moved on with recent everywhere.